app/controllers/concerns/verify_setup_completed.rb

```diff
@@ -4,14 +4,6 @@ module VerifySetupCompleted
 
   module_function
 
-  MANDATORY_CONFIGS = {
```

Comment:
These moved into `Settings::Mandatory` which seems like a better place, now this concern only needs to deal with displaying the banner if necessary.